HealthcareSystem/GraphicalEditor/Repository/MapObjectRepository.cs

```diff
@@ -0,0 +1,59 @@
+﻿using GraphicalEditor.Models;
+using GraphicalEditor.Repository.Intefrace;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GraphicalEditor.Repository
+{
+    public class MapObjectRepository : IMapObjectRepository
+    {
+        protected IEnumerable<MapObject> objects;
+        protected IRepository fileRepository;
+        protected MapObjectRepository() {
+            objects = fileRepository.LoadMap();
+        }
+
+        public MapObjectRepository(string path)
+        {
+        }
+
+        public bool Update(MapObject mapObject)
+        {
+
+            fileRepository = new FileRepository("test.json");
+            IEnumerable<MapObject> objects = fileRepository.LoadMap();
+            var ob = objects.FirstOrDefault(e => e.MapObjectEntity.Id == mapObject.MapObjectEntity.Id);
+            if (ob != null)
+            {
+                List<MapObject> savingList = new List<MapObject>();
+                foreach (MapObject m in objects)
+                {
+                    savingList.Add(m);
+                }
+                for (int i = 0; i < savingList.Count; i++)
+                {
+
+                    if (savingList[i].MapObjectEntity.Id == mapObject.MapObjectEntity.Id)
+                    {
+                        savingList[i].MapObjectEntity.MapObjectType = mapObject.MapObjectEntity.MapObjectType;
+                        savingList[i].MapObjectEntity.Description = mapObject.MapObjectEntity.Description;
+                        savingList[i].MapObjectMetrics = mapObject.MapObjectMetrics;                        
+                        //savingList[i].Rectangle = mapObject.Rectangle;  
+                        //savingList[i].MapObjectDoor = mapObject.MapObjectDoor;                          
```

Comment:
Why is this dead code here?

Reply:
that part we need to uncomment later

Reply:
Then please make comment above these lines. Something like this:
```sh

// Uncomment this if you want SOME-DESCRIPTION
// some line of code
// some line of code


```

We write code for our colleagues also. So if someone in the future comes, he will know why are these lines of code commented on and what they will do if they uncomment them.